lib/dotcom_web/views/alert_view.ex

```diff
@@ -177,7 +177,7 @@ defmodule DotcomWeb.AlertView do
         Timex.format!(updated_at, "{M}/{D}/{YYYY}")
       end
 
-    time = Timex.format!(updated_at, "{h12}:{m} {AM} {Zabbr}")
+    time = Timex.format!(updated_at, "{h12}:{m} {AM} {Zabbr}") |> String.trim()
```

Comment:
This fixes that failing test where there was trailing whitespace after `AM`. I believe that happens when the timezone isn't present.